boards/esp32-wrover-kit/Kconfig

```diff
@@ -22,5 +22,12 @@ config BOARD_ESP32_WROVER_KIT
     select HAS_SDCARD_SPI
 
     select HAVE_ILI9341
+    select HAVE_RIOT_CI_BUILD_ESP_HW_COUNTER
+    select HAVE_RIOT_CI_BUILD_ESP_I2C_HW
+    select MODULE_ESP_IDF_HEAP if TEST_KCONFIG && RIOT_CI_BUILD
+    select MODULE_ESP_LOG_STARTUP if TEST_KCONFIG && RIOT_CI_BUILD
+    select MODULE_ESP_LOG_TAGGED if TEST_KCONFIG && RIOT_CI_BUILD
+    select MODULE_ESP_QEMU if TEST_KCONFIG && RIOT_CI_BUILD
+    select MODULE_ESP_SPI_RAM if TEST_KCONFIG && RIOT_CI_BUILD
```

Comment:
I thought we were getting rid of these?

Reply:
I thought we were too.. I guess my dropped commit didn't contain all the changes... sorry.